requirements.txt

```diff
@@ -10,3 +10,5 @@ tqdm
 h5py
 structlog
 pytorch-lightning==0.8.5
+transformers
```

Comment:
This will always install it right?

Reply:
yeah because of the tests

Reply:
Should be in test-requirements then

Reply:
also, I thought since we provide a wrapper for transformers (it is part of the library similar to PL), so we should add it to the dependencies